packages/apps/plugins/plugin-navtree/src/components/TreeViewContainer.tsx

```diff
@@ -243,6 +251,7 @@ export const TreeViewContainer = ({
               onOver={handleOver}
               onDrop={handleDrop}
               popoverAnchorId={popoverAnchorId}
+              renderPresence={renderPresence}
```

Comment:
`NavTree` was meant to be less opinionated than this, though since it’s an intermediate component I have no principled objection to this approach. Something to consider if we want to promote it as an ecosystem component.